packages/jest-snapshot/src/TestSnapshot.js

```diff
@@ -9,7 +9,7 @@
 
 const fs = require('fs');
 
-class Snapshot {
+class TestSnapshot {
```

Comment:
Let's call it SnapshotFile instead. Sorry about the thrash, but it made sense when we talked about it in our meeting :)